packages/ui/src/machines/authenticator/actions.ts

```diff
@@ -172,7 +178,7 @@ export const setUser = assign({
      * @migration event.data was the fallback here,
      *  setting the entire event.data as user
      */
-    return { user: event.data?.userId };
+    return { ...event.data };
```

Comment:
Can we remove commented out code lines 173-180?

Reply:
I think this is not completely fixed yet, leaving the @migration comment so that I know to look at this